backend/app.py

```diff
@@ -61,36 +67,61 @@ def ontocall(path) -> [{}]:
     print(f'ontocall: {api_path}\n{url}')
 
     try:
-        response: List[Dict] = requests.get(url, headers=headers)
+        response = requests.get(url, headers=headers)
         response.raise_for_status()
-        json = response.json()
+        json: Dict = response.json()
         if 'data' in json:
             data = json['data']
         else:
             data = json
         if 'properties' in data:
-            data = data['properties'] # e.g., http://127.0.0.1:8000/ontocall?path=objects/OMOPConceptSet/729489911
+            data = data['properties']  # e.g., http://127.0.0.1:8000/ontocall?path=objects/OMOPConceptSet/729489911
             data['rid'] = json['rid']
     except BaseException as err:
         print(f"Unexpected {err=}, {type(err)=}")
         return {'ERROR': str(err)}
 
     return data
 
-    # noinspection PyTypeChecker
-    if path == 'objectTypes':
-        # data = json['data']
-        print(data)
-        return data
-        api_names = sorted([
-            t['apiName'] for t in data if t['apiName'].startswith('OMOP')])
-        return api_names
-    if path.startswith('objectTypes/'):
-        return json
-    if path.startswith('objectTypes/'):
-        return json
-
-    return {'valid but unhandled path': path, 'json': json}
+    # TODO: @siggie: This code was unreachable because of `return data` above, so i commented out
+    # if path == 'objectTypes':
+    #     # data = json['data']
+    #     print(data)
+    #     return data
+    #     api_names = sorted([
+    #         t['apiName'] for t in data if t['apiName'].startswith('OMOP')])
+    #     return api_names
+    # if path.startswith('objectTypes/'):
+    #     return json
+    # return {'valid but unhandled path': path, 'json': json}
+
+
+@app.put("/datasets/vocab")
+def vocab_update():
+    """Update vocab dataset"""
+    pass
+
+
+# TODO: figure out where we want to put this. models.py? Create route files and include class along w/ route func?
+class CsetsUpdate(BaseModel):
```

Comment:
@Sigfried Couldn't get it to work w/out using pydantic and this. I wouldn't consider that "fast" (a la FastAPI), but that's fine.

Reply:
Also:
> # TODO: figure out where we want to put this. models.py? Create route files and include class along w/ route func?